go/vt/sqlparser/parse_test.go

```diff
@@ -1433,6 +1433,15 @@ var (
 		}, {
 			input:  "analyze table a, b, c",
 			output: "analyze table a, b, c",
+		}, {
+			input:  "prepare stmt from 'select sqrt(pow(?,2) + pow(?,2)) as hypotenuse'",
+			output: "prepare stmt from 'select sqrt(pow(?,2) + pow(?,2)) as hypotenuse'",
+		}, {
+			input:  "execute stmt using @a, @b",
+			output: "execute stmt using @a, @b",
+		}, {
+			input:  "deallocate prepare stmt",
+			output: "deallocate prepare stmt",
```

Comment:
Is it easy to add support for `DROP PREPARE stmt`, too, while you're in here? That's another syntax form for this that MySQL supports. 

https://dev.mysql.com/doc/refman/8.0/en/deallocate-prepare.html